build/test-replacements.js

```diff
@@ -214,3 +231,12 @@ module.exports['test-stream-pipe-cleanup-pause.js'] = [
     'var '
   ]
 ]
+module.exports['test-stream2-readable-empty-buffer-no-eof.js'] = [[
+  /let /g,
```

Comment:
there is a single let, this might actually be a style error in the original test